Alicerce/Sources/Extensions/Collection.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Collection.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 18/12/2016.
+//  Copyright © 2016 Mindera. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - Array
+
+func -<Key: Hashable, Value: AnyObject>(left: [[Key : Value]], right: [[Key : Value]]) -> [[Key : Value]] {
+    var final = left
+    
+    for (index, element) in right.enumerated() {
+        final[index] = (left[index] - element)
+    }
+    
+    return final
+}
+
+// MARK: - Dictionary
+
+func -<Key: Hashable, Value: AnyObject>(left: [Key : Value], right: [Key : Value]) -> [Key : Value] {
+    var final = left
+    
+    for (key, value) in right {
```

Comment:
Just to be clearer I would perhaps rename this `value` to `lValue` or `leftValue`, and the `val` below `rValue` or `rightValue` 😉